mne/epochs.py

```diff
@@ -1560,6 +1572,16 @@ def crop(self, tmin=None, tmax=None, include_tmax=True, verbose=None):
         self._set_times(self.times[tmask])
         self._raw_times = self._raw_times[tmask]
         self._data = self._data[:, :, tmask]
+
+        # Adjust rejection period
+        if self.reject_tmin is not None and self.reject_tmin < self.tmin:
+            warn('reject_tmin is not in epochs time interval. reject_tmin is '
+                 'set to epochs.tmin')
```

Comment:
Do we actually need to warn for this? I think `logger.info` is probably more appropriate when cropping.

Reply:
I've changed it to info